tools/bin/release_version.sh

```diff
@@ -37,3 +37,12 @@ VERSION=$NEW_VERSION SUB_BUILD=PLATFORM ./gradlew publish
 docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
 VERSION=$NEW_VERSION ./tools/bin/publish_docker.sh
 echo "Completed building and publishing PLATFORM..."
+
+echo "Packaging and publishing HELM CHARTS version $NEW_VERSION for git revision $GIT_REVISION..."
```

Comment:
Cufious, why release the subcharts in a spot different from the main helm charts?

Reply:
I've already covered it during our call, made it that way since we're packaging main chart using dependencies that were already published, I can make it use local dependencies but don't know how it will work and probably this approach will require clonning of the remote git repo in order to deploy it.

Reply:
Yeah. What I'm asking is why do this in this spot instead of in the Github Action? It seems like this can be combined with the above action. It's somewhat confusing today since the subcharts are released at a different spot from the main chart so it's not obvious we have to release the subchart before the main charts.

Reply:
I see, already made an action for it, will push updated workflow

Reply:
@davinchia updated + added workflow for publishing charts manually(separately from the OSS releases)